RevenueCatUI/View+PresentPaywall.swift

```diff
@@ -16,6 +16,15 @@ import SwiftUI
 
 #if !os(macOS) && !os(tvOS)
 
+/// Presentation options to use with the View modifiers ``presentPaywallIfNeeded``.
+///
+/// ### Related Articles
+/// [Documentation](https://rev.cat/paywalls)
+public enum PaywallPresentationMode {
+    case sheet
+    case fullScreen
+}
```

Comment:
Nit: our Swift style guide requires spaces around types:
```suggestion

    case sheet
    case fullScreen
    
}
```